templates/cube/microcerts.html

```diff
@@ -210,17 +210,18 @@ <h2>Microcertifications</h2>
             </div>
           </td>
           <td class="p-table--cube__badge-column" aria-label="Badge">
-              <img
-                class="p-table--cube__badge"
-                src="{{ module.badge.url }}"
-                alt="Badge for {{ module.name }}"
-                {% if module.status == 'passed' %}
-                style="margin-left: -0.2rem;"
-                {% endif %}
-              />
+            <img
+              class="p-table--cube__badge"
+              src="{{ module.badge.url }}"
+              alt="Badge for {{ module.name }}"
+              {% if module.status == 'passed' %}
+              style="margin-left: -0.2rem;"
+              {% endif %}
```

Comment:
I don't understand, should this switch out the image? This is what I see:
![image](https://user-images.githubusercontent.com/1413534/119485820-70f36f00-bd4f-11eb-8cee-25f8e7b8b674.png)

Reply:
No, I'll double check with Joao but I think as we are fake passing the microcerts the "passed" microcert isn't being fetched from Badgr so (that's why it's hard to QA.) But I remember having to add this at the time of building as the dimensions of the passed badge were slightly different.